massa-node/src/tests/testnet.toml

```diff
@@ -0,0 +1,114 @@
+# Testing configuration used in unit tests
+# Do not use it for anything else
+
+version = "TEST.4.2"
```

Comment:
You should change the version to deve to make sure that it won't interact with the testnet.

And you should rename the file in order yet again to avoid confusion with The Testnet

Reply:
@AureliaDolo all is done in the latest commit :smile_cat: